projects/m1/033-shipping-calculator/js/index.js

```diff
@@ -0,0 +1,29 @@
+let ordersArray = [];
+
+function startInput() {
+   
+    let newOrder = "";
+
+    while (newOrder !== "n") {
+        newOrder = prompt("Inserisci un articolo nel carrello (inserisci n per terminare):");
+
+        if (newOrder !== "n") {
+            ordersArray.push(newOrder);
+        }
+    }
+}
+
+function definitiveCost(totalOrders) {
+    let basicCost = 10.99;
+    let otherCost = 2.99;
+    let totalCost = basicCost + (otherCost * totalOrders);
```

Comment:
This solution **does not seem correct** to me.
I recommend _re-reading the exercise_ trace and retrying to solve the problem.